tidy3d/components/validators.py

```diff
@@ -396,3 +396,21 @@ def freqs_not_empty(cls, val):
         return val
 
     return freqs_not_empty
+
+
+def validate_mode_plane_radius(mode_spec: "ModeSpec", plane: "Box", msg_prefix: str):  # noqa: F821
```

Comment:
maybe `ms_prefix: str =''` in case we dont want to require it, very minor though.